src/generic/generic-component-list-item/generic-component-list-item.scss

```diff
@@ -167,12 +167,4 @@
       text-decoration: none;
     }
   }
-
```

Comment:
In fact, it should have been the other way around. In mobile-first, you should never use `ecl-media-breakpoint-down`, but always use `ecl-media-breakpoint-up`. The description in the ticket was wrong. Sorry for that